### ISSUE TITLE
Overzealous filters

### ISSUE THREAD

**Issue by Nateowami** [Nateowami/Backloader#16]
The default filters block `https://facebook.github.io/`. The filter responsible is `*facebook.*` I think there ought to be a way to match the main domain and all sub domains without blocking domains that end with the same name, but I can't think of what the right syntax would be. Perhaps something like `*.?facebook.*` Seems that could be problematic.

For this particular rule, would there by any problem changing it to `*facebook.com`? That would still block domains that end with `facebook.com` of course. Are there any Facebook TLDs that need to be blocked, other than `.com`, that would make this not work?

**Comment by JoshTheDerf** [comment]
I believe there was another facebook TLD, and yes, we should probably just block TLDs in the default filters. Making the Regex more complicated is something that should be avoided.

**Comment by Nateowami** [comment]
In other words, we should have two separate filters, `*facebook.com` and 
`*facebook.some_other_tld`. Is that what you're saying?

I agree making the regex more complicated is not a good idea. Perhaps we 
could make it that `*.facebook.com` also matches `facebook.com`?

**Comment by JoshTheDerf** [comment]
*facebook.com would match that, right?

And yes. I think facebook's like buttons or ad thingies are on a different TLD. Still can't find it though. :/

**Comment by Nateowami** [comment]
Yes, it would match both `facebook.com` and `something.facebook.com`, but also `somethingelsefacebook.com`.

Looks like it's `connect.facebook.net` that serves the like buttons. I can split it into two rules.

**Comment by JoshTheDerf** [comment]
Ah, that was it.
